flanker/addresslib/corrector.py

```diff
@@ -30,177 +30,191 @@ def suggest(word, cutoff=0.77):
         return guess[0]
     return word
 
-
 MOST_COMMON_DOMAINS = [
     # mailgun :)
     'mailgun.net',
-    # big esps
+    # The Top 200 domains from 30mm open events gathered during the
+    # last 30 days prior to 8/22/2018
+    '126.com',
     '163.com',
+    'account.tradeshift.com',
     'aim.com',
+    'airwallex.com',
     'alice.it',
-    'aol.co.uk',
+    'ameritech.net',
+    'anz.com',
     'aol.com',
+    'apple.com',
+    'asu.edu',
     'att.net',
-    'azet.sk',
+    'bain.com',
     'bell.net',
     'bellsouth.net',
+    'berkeley.edu',
     'bigpond.com',
     'bigpond.com.au',
     'bigpond.net.au',
+    'bk.ru',
     'bluewin.ch',
-    'blueyonder.co.uk',
     'bol.com.br',
     'btinternet.com',
-    'btopenworld.com',
-    'cableone.net',
-    'centrum.sk',
     'centurylink.net',
-    'centurytel.net',
+    'cfl.rr.com',
+    'charter.com',
     'charter.net',
+    'citi.com',
+    'colemanrg.com',
+    'columbia.edu',
     'comcast.net',
+    'compass.com',
+    'cornell.edu',
     'cox.net',
-    'cs.com',
+    'cpp.edu',
+    'cytonn.com',
+    'deloitte.com',
+    'det.nsw.edu.au',
+    'docomo.ne.jp',
     'earthlink.net',
-    'email.com',
-    'email.cz',
-    'email.it',
+    'edumail.vic.gov.au',
+    'eim.ae',
+    'email.arizona.edu',
     'embarqmail.com',
+    'emirates.net.ae',
     'excite.com',
-    'fastwebnet.it',
+    'exploretrip.com',
+    'ext.uber.com',
+    'ezweb.ne.jp',
     'free.fr',
-    'freemail.hu',
     'freenet.de',
     'frontier.com',
     'frontiernet.net',
     'fuse.net',
+    'g.austincc.edu',
+    'ge.com',
     'gmail.com',
     'gmx.at',
-    'gmx.ch',
     'gmx.com',
     'gmx.de',
     'gmx.net',
     'google.com',
     'googlemail.com',
+    'gs.com',
+    'guest.booking.com',
     'hanmail.net',
-    'home.nl',
-    'hotmail.be',
     'hotmail.ca',
     'hotmail.co.jp',
     'hotmail.co.nz',
     'hotmail.co.uk',
     'hotmail.com',
-    'hotmail.com.ar',
     'hotmail.com.au',
     'hotmail.de',
     'hotmail.es',
     'hotmail.fr',
-    'hotmail.gr',
     'hotmail.it',
-    'hotmail.nl',
-    'hotmail.no',
-    'hotmail.se',
-    'hughes.net',
+    'i.softbank.jp',
     'icloud.com',
     'iinet.net.au',
-    'inbox.lv',
     'inbox.ru',
-    'interia.pl',
+    'indeedemail.com',
+    'internationalsos.com',
+    'internode.on.net',
+    'iprimus.com.au',
+    'jhu.edu',
+    'jtbap.com',
     'juno.com',
+    'kw.com',
     'laposte.net',
     'libero.it',
+    'limetray.com',
     'list.ru',
-    'live.be',
     'live.ca',
     'live.co.uk',
     'live.com',
-    'live.com.ar',
     'live.com.au',
-    'live.com.mx',
-    'live.de',
-    'live.dk',
     'live.fr',
-    'live.it',
     'live.nl',
-    'live.no',
-    'live.se',
+    'lynk.co.in',
     'mac.com',
     'mail.com',
     'mail.ru',
+    'marketplace.amazon.com',
+    'marriott.com',
+    'mavs.uta.edu',
+    'mchsi.com',
+    'mckinsey.com',
     'me.com',
-    'microsoft.com',
```

Comment:
pretty surprised to see this fall off the list entirely

Reply:
same, but it's not even in the top 200

https://gist.github.com/Cerberus98/8e723091e578c4b1b8215ff9064d02f7